### ISSUE TITLE
aimbot not working

### ISSUE THREAD

**Issue by nexo1221** [danielkrupinski/Osiris#3994]
guys aimbot section doesn't work at all

**Comment by Xsintashi** [comment]
works

**Comment by GiaNTizmO** [comment]
1) Did you make some changes in source code?
2) Does you use latest version of Osiris?
3) Can you make short video with turning on aimbob? Mb you make something wrong...

**Comment by nexo1221** [comment]
No, i did not modify anything, and i used the laat version of Osiris. I
will make an video and send it to You.

lun., 27 mar. 2023, 22:52 GiaNTizmO ***@***.***> a scris:

**Comment by nexo1221** [comment]
2023-03-28 07-29-52.mkv
<https://drive.google.com/file/d/1EIsPmJooT9kgkiHz9M518la5RepZB3sM/view?usp=drive_web>
i did not make nothing wrong, i tried to make some adjustments to the
setting of the aimbot but nothing works and im pressing my button for it to
activate

În mar., 28 mar. 2023 la 07:22, Full Moon001 ***@***.***> a scris:

**Comment by nexo1221** [comment]
i haved used osiris before but aimbot worked now it wont
